test/test_transformers.py

```diff
@@ -1454,6 +1454,25 @@ def test_is_causal_gpu(self):
         device = 'cuda'
         self.is_causal_kernels(["math", "meff"], device)
 
+    def test_mask(self):
+        def ones_tensor(*shape):
+            return torch.ones(shape, dtype=torch.float32)
+        S, L, E, H = 1, 2, 4, 1
+        qkv = ones_tensor(S, L, E)
+
+        mha = nn.MultiheadAttention(E, H)
+        mha.in_proj_weight = Parameter(torch.ones((E * 3, E)))
+        mha.out_proj.weight = Parameter(torch.ones((E, E)))
+        expected = torch.ones(size=(S, L, E)) * 16
```

Comment:
nit: is this used?
Are is this test just to ensure that kpm and am can't both be set? if so I think the test name should somewhat reflect that